tests/test_fgnhg.py

```diff
@@ -0,0 +1,947 @@
+import os
+import re
+import time
+import json
+import pytest
+
+from swsscommon import swsscommon
+
+def create_entry(tbl, key, pairs):
+    fvs = swsscommon.FieldValuePairs(pairs)
+    tbl.set(key, fvs)
+    time.sleep(1)
+
+
+def create_entry_tbl(db, table, separator, key, pairs):
+    tbl = swsscommon.Table(db, table)
+    create_entry(tbl, key, pairs)
+
+def remove_entry_tbl(db, table, key):
+    tbl = swsscommon.Table(db, table)
+    tbl._del(key)
+    time.sleep(1)
```

Comment:
we should not use such sleep. it is causing the test to be unstable.

Reply:
I do see the sleep used across the board in the tests today, eg: https://github.com/Azure/sonic-swss/blob/master/tests/test_nhg.py#L117 . Without sleep the test case fails since it may not be enough time for the change to take effect. I do see that it can cause tests to be unstable since there is a possibility that 1s is not enough if the system is exceptionally slow. Is there a good alternative we are using today?

Reply:
You may use one of the `wait_*` functions

---
In reply to: [477487120](https://github.com/Azure/sonic-swss/pull/1315#discussion_r477487120) [](ancestors = 477487120)

Reply:
I see thanks, will use that